server/app/filters/LoggingFilter.java

```diff
@@ -24,14 +25,25 @@ public class LoggingFilter extends EssentialFilter {
   private static final Logger log = LoggerFactory.getLogger("loggingfilter");
   private final Config config;
 
-  /** @param exec This class is needed to execute code asynchronously. */
+  /**
+   * @param exec This class is needed to execute code asynchronously.
+   */
   @Inject
   public LoggingFilter(Executor exec, Clock clock, Config config) {
     this.exec = checkNotNull(exec);
     this.clock = checkNotNull(clock);
     this.config = config;
   }
 
+  private String statusWithOptionalRedirectLocation(Result result) {
+    int status = result.status();
+    if (status >= 300 && status < 400 && result.redirectLocation().isPresent()) {
```

Comment:
Is status here referring to http status code?

Reply:
That's correct!